packages/lit-dev-content/site/articles/article/lit-for-polymer-users.md

```diff
@@ -0,0 +1,1021 @@
+---
+title: Lit for Polymer users
+publishDate: 2022-08-23
+lastUpdated: 2020-08-23
+summary: Moving your code from Polymer to Lit.
+tags:
+  - migration
+  - webcomponents
```

Comment:
When you add a new tag, do you mind also adding a respective `tags/tagname.md` file? Here's an example "composition" tag which you can remove like you did with `light-dom.md`: 

https://github.com/lit/lit.dev/tree/main/packages/lit-dev-content/site/articles/tags

The thing these tags need to match is `childrenTag`, the other ones are human-readable things.

Reply:
Oops. Didn't even look, just assumed the tag list was generated. 

Hmm... No idea what real tags to use here. Maybe I'll just use #webcomponents for now?

Reply:
i think whatever for now is fine. We should come together later for the tag taxonomy (tagsonomy if you will) as they will be instrumental in the learn page's implementation

Reply:
Does the tags UI show up anywhere that I can test my changes?

Reply:
It will be more discoverable once we ship it. The LHS nav is currently `display: none` as well as the tags in the header.

But you can test if they've linked correctly by going to this link:

https://pr909-ac72e49---lit-dev-5ftespv5na-uc.a.run.app/articles/tags/webcomponents/

and you can see "all articles" here:

https://pr909-ac72e49---lit-dev-5ftespv5na-uc.a.run.app/articles/

Looks good to me!

Reply:
unhiding those elements (save for some things i've accidentally reverted in my last PR) it looks good:
<img width="1457" alt="image" src="https://user-images.githubusercontent.com/5981958/185722114-092876e6-6e11-43db-bd75-1820273156c8.png">

Reply:
consider making `users` capitalized `Users`?

Reply:
Hmm... Normally we use sentence case for titles. That looks a little strange on this title since Polymer is capitalized, but if we make a one-off change here, it's going to be even more confusing next time.